docs/06-contributors/99-maintainers/050-operations.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Operations Overview
+id: operations
+keywords: [operations]
+---
+
+## Coverage
+
+"Wing" encompases several projects/sites.
+This document covers operations for the following:
+
+- https://docs.winglang.io (https://github.com/winglang/docsite)
+- Wing (https://github.com/winglang/wing)
+- Wing Console (https://github.com/winglang/console)
+
+## Severity Levels
+
+Incidents are classified by severity level.
+The severity level is determined by the impact of the incident on user experience.
+
+| Severity | Description | Example |
+| - | - | - |
+| P0 | Extreme user interruption, no reasonable workaround | Unable to install Wing |
+| P1 | Major degradation of user experience, possible workarounds | Main build is broken in CI (cannot publish) |
+| P2 | The system is usable but has a minor issue. | Error message regression, new message is very confusing |
+
+## Response Process
+
+### Acknowledgement
+
+Acknowledgement signifies that someone is taking responsibility for the incident.
+This does not mean that the person acknowledging the incident is who will be resolving it, but they will at least be responsible for triage and communication.
+
+Depending on the issue type, the acknowledgement may be automated through tooling (e.g. BetterUptime).
+If not automated, the responder should create a thread in the #alert channel in Slack to make sure others are aware.
+
+### Response
```

Comment:
Perhaps we should note in the response section to document steps taken if possible. These might be helpful during post-mortem. (what actions have been taken to resolve the problem, who was reached out to at what point)